### PR TITLE
Add JSON-LD to events

### PR DESCRIPTION
**Pull request by Pelps12**
Jason take a look at the PR before I push

### CODE REVIEW

File: apps/events/src/pages/events/[id].tsx [hunk 1]
```diff
@@ -132,8 +125,8 @@ const Event: NextPage<InferGetServerSidePropsType<typeof getServerSideProps>> =
 				openGraph={{
 					title: `${props?.meta?.title}` ?? 'Event',
 					description: `Event Details of ${props?.meta?.title}`,
-					url: `https://${env.NEXT_PUBLIC_URL}/seller/${props.meta?.id}`,
-					type: 'profile',
+					url: `${env.NEXT_PUBLIC_URL}/seller/${props.meta?.id}`,
```
Comment (jasonappah) [inline apps/events/src/pages/events/[id].tsx]: Is it intentional to use /seller here?

File: apps/events/src/pages/events/[id].tsx [hunk 2]
```diff
@@ -146,6 +139,24 @@ const Event: NextPage<InferGetServerSidePropsType<typeof getServerSideProps>> =
 					"Join us for an exciting event that promises to be a memorable experience. While the event details are not provided by the organizer, you can expect a day filled with fun, entertainment, and engagement. Stay tuned for updates and surprises as we get closer to the event date. Don't miss out on this fantastic opportunity to connect with like-minded individuals and enjoy a day of excitement. Save the date, and we look forward to sharing more information soon!"
 				}
 			/>
+
+			<EventJsonLd
+				name={props?.meta?.title}
+				startDate={props?.meta?.startTime}
+				endDate={props?.meta?.endTime}
+				location={props?.meta?.location}
+				url={`${env.NEXT_PUBLIC_URL}/seller/${props.meta?.id}`}
```
Comment (jasonappah) [inline apps/events/src/pages/events/[id].tsx]: Same as previous comment

**Review by jasonappah — CHANGES_REQUESTED** [review]
Everything looks fine minus the few comments i added

File: apps/events/src/pages/events/[id].tsx (hunk 1 above)
Comment (Pelps12) [reply apps/events/src/pages/events/[id].tsx]: No. I'll fix that now

**Review by jasonappah — APPROVED** [review]

**Pull request merged**